app/initializers/admin.js

```diff
@@ -1,12 +1,15 @@
 import AdminStore from 'ember-admin/stores/admin';
 
-export default {
-  name: 'admin',
-  initialize: function(container, app) {
+export function  initialize(container, app) {
```

Comment:
The header for this function should be rewritten as `(registry, app)` as the `container` is no longer passed in as of Ember 1.11(ish)